src/Classes/Utils/Messages.ts

```diff
@@ -0,0 +1,7 @@
+export class Messages {
+    public static deprecationMessage(oldProperty: string, newProperty: string) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [expected call-signature: 'deprecationMessage' to have a typedef](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5139697)